src/ape/api/providers.py

```diff
@@ -316,29 +323,30 @@ def send_transaction(self, txn: TransactionAPI) -> ReceiptAPI:
     @abstractmethod
     def get_contract_logs(
         self,
-        address: Union[AddressType, List[AddressType]],
-        abi: Union[EventABI, List[EventABI]],
+        addresses: Union[AddressType, List[AddressType]],
+        abis: Union[EventABI, List[EventABI]],
         start_block: Optional[int] = None,
         stop_block: Optional[int] = None,
         block_page_size: Optional[int] = None,
-        event_parameters: Optional[Dict] = None,
+        search_topics: Optional[Dict] = None,
     ) -> Iterator[ContractLog]:
         """
-        Get all logs matching the given set of filter parameters.
+        Get logs from contracts.
 
         Args:
-            address (``AddressType``): The contract address that defines the logs.
-            abi (``EventABI``): The event of interest's ABI.
-            start_block (Optional[int]): Get events that occurred
-              in blocks after the block with this ID.
-            stop_block (Optional[int]): Get events that occurred
-              in blocks before the block with this ID.
-            block_page_size (Optional[int]): Use this parameter to adjust
-              request block range sizes.
-            event_parameters (Optional[Dict]): Filter by event parameter values.
+            addresses (Union[AddressType, List[AddressType]): Contract addresses.
+            abis (Union[EventABI, List[EventABI]]): Event ABIs.
```

Comment:
- [ ] Refactor so that its a `topic_query: Dict[EventABI, Dict]` ! To better support OR-ing